swagger-apis/metrics-collection-platform/2.0.0-beta.yml

```
@@ -1996,6 +1995,16 @@ components:
                 Regras:
                   - Preencher com o valor do campo "cancellation.reason"
               type: string
+            nfcPayment:
+              description: |
+                As detentoras devem armazenar a informação e correlacioná-la com o consentimento e o pagamento originado. Replicar o valor presente do header “x-bcb-nfc” do endpoint POST /consents/{consentId}/authorise 
```

Meus 2 centavos de sugestão

```suggestion
                As detentoras devem armazenar a informação e correlacioná-la com o consentimento e o pagamento correspondente. Deve-se também replicar o valor presente no header “x-bcb-nfc” do endpoint POST /consents/{consentId}/authorise.
```